tests/integration/optimizers/optimize.py

```diff
@@ -0,0 +1,42 @@
+from fashion.data import get_data
+from fashion.evaluation import index_document_generator, evaluation_document_generator
+
+from jina.optimizers import OptunaOptimizer, EvaluationCallback
+from jina.optimizers.flow_runner import FlowRunner
+
+DATA_DIRECTORY = "data"
+
+
+def main():
```

Comment:
feels weird to have a `main` under `tests`

Reply:
This is temporary